DataAccess/Models/FundRequest.cs

```diff
@@ -3,29 +3,56 @@
 using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
+using System.ComponentModel.DataAnnotations;
 
 namespace DataAccess.Models
 {
     public class FundRequest
     {
         public int FundRequestID { get; set; }
+
+        [Required(ErrorMessage = "Grade is required.")]
+        [Range(1, 12, ErrorMessage = "Grade must be between 1 and 12.")]
         public byte Grade { get; set; }
+
```

Comment:
Hey @Penelope32. You made a mistake on grade. It should be average mark(grade), not school grades. Can you be able to fix that when you're done with the test?